preprocessors/openstreetmap/osm_service.py

```diff
@@ -543,3 +550,14 @@ def get_coordinates(content):
     """
     if 'coordinates' in content.keys():
         return content['coordinates']
+
+
+def scale_distance(content):
```

Comment:
It looks like you're adding distance to the request parameters...first off, you need to specify units for this in the code. I'm assuming this is in meters, which would be perfect. Second, adding new fields to the request from the extension is much more involved than just this...currently this wouldn't be valid and I don't believe we can even get this data from the embedded maps at this point. I'll get discussion on this added to the next tech arch meeting if not before then. In the meantime, with this feature we'll need to be very careful in how we choose the default value (here, I believe this would be within a square of 400 m since the sides are each 200 m from the center, right?) since this default is what will end up being used for the foreseeable future.

Reply:
Yes, the distance is in metres, with the map area being 400sqm as rightly said. Your concerns are well noted.